docs/content/en/schemes/refresh.md

```diff
@@ -77,6 +77,7 @@ auth: {
         // type: 'Bearer'
       },
       refreshToken: {
+        global: true,
```

Comment:
Refresh token doesn't have `global` option. Can you move this to `token` options?